init-localstack.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -x
+pip install awscli-local
+awslocal s3 mb s3://$AWS_BUCKET_NAME
+awslocal s3api put-bucket-acl --bucket $AWS_BUCKET_NAME --acl public-read
```

Comment:
If we want to emulate what it's like production, we use private as the bucket policy, then individual objects override it via public-read when necessary

Reply:
Thanks for the pointer, I have made the change